util/types/convert.go

```diff
@@ -97,7 +97,7 @@ func convertIntToInt(val, lowerBound, upperBound int64, tp byte) (converted int6
 	return
 }
 
-func convertToInt(val interface{}, target *FieldType) (converted int64, err error) {
+func convertToInt(val interface{}, target FieldType) (converted int64, err error) {
```

Comment:
I see FieldType variable is only used for read variable, also in ColumnInfo struct with no pointer FieldType,  so i think it is better to use a none-pointer variable here.